Jenkins compile error due to upstream 0.23.0-SNAPSHOT changes

```
...
[ERROR] <https://amplab.cs.berkeley.edu/jenkins/job/cannoli/HADOOP_VERSION=2.3.0,SCALAVER=2.10,SPARK_VERSION=2.0.0,label=centos/ws/cli/src/main/scala/org/bdgenomics/cannoli/cli/Freebayes.scala>:86: error: not enough arguments for constructor VCFOutFormatter: (conf: org.apache.hadoop.conf.Configuration)org.bdgenomics.adam.rdd.variant.VCFOutFormatter.
[ERROR] Unspecified value parameter conf.
[ERROR]     implicit val uFormatter = new VCFOutFormatter
[ERROR]                               ^
[ERROR] <https://amplab.cs.berkeley.edu/jenkins/job/cannoli/HADOOP_VERSION=2.3.0,SCALAVER=2.10,SPARK_VERSION=2.0.0,label=centos/ws/cli/src/main/scala/org/bdgenomics/cannoli/cli/Freebayes.scala>:103: error: could not find implicit value for parameter xFormatter: org.bdgenomics.adam.rdd.OutFormatter[org.bdgenomics.adam.models.VariantContext]
[ERROR]     val output: VariantContextRDD = input.pipe[VariantContext, VariantContextRDD, BAMInFormatter](freebayesCommand)
[ERROR]                                                                                                  ^
[ERROR] <https://amplab.cs.berkeley.edu/jenkins/job/cannoli/HADOOP_VERSION=2.3.0,SCALAVER=2.10,SPARK_VERSION=2.0.0,label=centos/ws/cli/src/main/scala/org/bdgenomics/cannoli/cli/SnpEff.scala>:89: error: not enough arguments for constructor VCFOutFormatter: (conf: org.apache.hadoop.conf.Configuration)org.bdgenomics.adam.rdd.variant.VCFOutFormatter.
[ERROR] Unspecified value parameter conf.
[ERROR]     implicit val uFormatter = new VCFOutFormatter
[ERROR]                               ^
[ERROR] <https://amplab.cs.berkeley.edu/jenkins/job/cannoli/HADOOP_VERSION=2.3.0,SCALAVER=2.10,SPARK_VERSION=2.0.0,label=centos/ws/cli/src/main/scala/org/bdgenomics/cannoli/cli/SnpEff.scala>:104: error: could not find implicit value for parameter xFormatter: org.bdgenomics.adam.rdd.OutFormatter[org.bdgenomics.adam.models.VariantContext]
[ERROR]     val output: VariantContextRDD = input.pipe[VariantContext, VariantContextRDD, VCFInFormatter](snpEffCommand)
[ERROR]                                                                                                  ^
[ERROR] four errors found
```

--- a/cli/src/main/scala/org/bdgenomics/cannoli/cli/Freebayes.scala
+++ b/cli/src/main/scala/org/bdgenomics/cannoli/cli/Freebayes.scala
@@ -83,7 +83,7 @@ class Freebayes(protected val args: FreebayesArgs) extends BDGSparkCommand[Freeb
     val input: AlignmentRecordRDD = sc.loadAlignments(args.inputPath, stringency = stringency)
 
     implicit val tFormatter = BAMInFormatter
-    implicit val uFormatter = new VCFOutFormatter
+    implicit val uFormatter = new VCFOutFormatter(sc.hadoopConfiguration)
 
     val freebayesCommand = if (args.useDocker) {
       Seq("docker",

--- a/cli/src/main/scala/org/bdgenomics/cannoli/cli/SnpEff.scala
+++ b/cli/src/main/scala/org/bdgenomics/cannoli/cli/SnpEff.scala
@@ -86,7 +86,7 @@ class SnpEff(protected val args: SnpEffArgs) extends BDGSparkCommand[SnpEffArgs]
     val input: VariantContextRDD = sc.loadVcf(args.inputPath, stringency)
 
     implicit val tFormatter = VCFInFormatter
-    implicit val uFormatter = new VCFOutFormatter
+    implicit val uFormatter = new VCFOutFormatter(sc.hadoopConfiguration)
 
     val snpEffCommand = if (args.useDocker) {
       Seq("docker",

